### DISCUSSION TITLE
imac 12.2

### DISCUSSION THREAD

**Discussion by Computerclub-Plauen** [community/community#69581] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

imac 12.2 sonoma oclp 1.0.1 bluetooth not working

**Comment by Akash1134** [comment]
Hi @Computerclub-Plauen ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
